working/macros/feature-specification.md

```diff
@@ -871,23 +891,32 @@ Each argument in the metadata annotation for the macro application is converted
 to a form that the corresponding constructor on the macro class expects, which
 it specifies through parameter types:
 
-*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, or `String`
-    (or the nullable forms of any of those), then the argument expression must
-    be a Boolean, number, null, or string literal. Number literals may be
-    negated. String literals may not contain any interpolation, but may be
-    adjacent strings.
+*   If the parameter type is `bool`, `double`, `int`, `Null`, `num`, `String`,
+    `List`, or `Map`, (or the nullable forms of any of those), then the argument
+    expression must be a Boolean, number, null, string, list, or map literal.
+
+    * Number literals may be negated.
+    * String literals may not contain any interpolation, but may be adjacent
```

Comment:
How about raw Strings? (Multi-line is probably unnecessary to specify.)

Reply:
Should I clarify raw strings? I don't know if there is any consideration that makes them different from any other String literal.

Reply:
I have no strong opinion. I just put raw strings on the same level as adjacent strings, which are commented on.

Reply:
I added it, why not 🤷‍♂️